jsx/Filter.js

```diff
@@ -77,6 +81,19 @@ class Filter extends Component {
     }, []);
   }
 
+  renderActions() {
```

Comment:
i pushed some commits to this PR before your recent rebase that removed this function in Filter.js that I think you've now reverted. i don't think renderActions() belongs here unless I am missing something?

Reply:
No, I think I just forgot to pull the changes